src/main/java/com/digitalasset/testing/ledger/SandboxRunner.java

```diff
@@ -53,9 +53,9 @@ public SandboxRunner(
     commands.add("-p");
     commands.add(sandboxPort.toString());
     commands.add(useWallclockTime ? "-w" : "-s");
-    if (testModule.isPresent() && testScenario.isPresent()) {
+    if (testModule.isPresent() && testStartScript.isPresent()) {
       commands.add("--scenario");
-      commands.add(String.format("%s:%s", testModule.get(), testScenario.get()));
+      commands.add(String.format("%s:%s", testModule.get(), testStartScript.get()));
```

Comment:
This won't work any more. Maybe you had the same idea to use the old parameter for daml script?